DESCRIPTION

```diff
@@ -0,0 +1,11 @@
+Package: geomstats
+Type: Package
+Title: geomstats for R

```

Comment:
NIT: `Geomstats for R`

Reply:
@lundjack just missing the following correction: `Geomstats for R` instead of `geomstats for R`